test/fixtures/logging_app.dart

```diff
@@ -0,0 +1,37 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:developer';
+import 'dart:io';
+
+// Allow a test driver to communicate with this app through the controller.
+final Controller controller = new Controller();
+
+void main() {
+  log('starting app');
+  print('starting app');
```

Comment:
can the print calls be removed?

Reply:
I'd like to keep them just for the sake of making it easier to debug this app in some circumstances.